### ISSUE TITLE
Clerk Authentication Failing in Both Development and Production Environments

### ISSUE THREAD

**Issue by sahil-idk** [sahil-idk/opensource.build#7]
![image](https://github.com/sahil-idk/opensource.build/assets/141494404/5c8c03e8-b65c-4f35-bb13-0ea9ce4d2c96)

I just now came across this 
Should we opt for other authentication services such as Lucia Auth or some other authentication toolkit ?
@Sahil-Sharma-23 

Pls feel free to comment

**Comment by Sahil-Sharma-23** [comment]
> ![image](https://private-user-images.githubusercontent.com/141494404/346314848-5c8c03e8-b65c-4f35-bb13-0ea9ce4d2c96.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0NDQ5MDQsIm5iZiI6MTcyMDQ0NDYwNCwicGF0aCI6Ii8xNDE0OTQ0MDQvMzQ2MzE0ODQ4LTVjOGMwM2U4LWI2NWMtNGYzNS1iYjEzLTBlYTljZTRkMmM5Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwOFQxMzE2NDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYmFmMTVmNmZhMWI2ZDY2ODRiMmRiZjBhZjE2ODM5YWM2N2ZkYjJmNmQ1NjQyZDdkMzk5NWQ5MTM4ZWUwMGMyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.l7Rw1ZhFyI2oYKWPj3gpNej_iIZda34nkm1d4XpXXiE)
> 
> I just now came across this Should we opt for other authentication services such as Lucia Auth or some other authentication toolkit ? @Sahil-Sharma-23
> 
> Pls feel free to comment

Yes. Instead of fully managed auth providers, we can use something like NextAuth.js to have much more control over the authentication process.
Check the below doc link to get started.
https://next-auth.js.org/getting-started/example

**Comment by sahil-idk** [comment]
The problem is that in next auth it has very minimal interface and also has build fails after a while , lucia on other hand gives flexibility of interface but the code is more . i personally had build fails in nextauth and also other projects made by yt creators was same on their deployed app

**Comment by Sahil-Sharma-23** [comment]
In that case, we can then move ahead with lucia auth.
